subscriptions/views.py

```diff
@@ -60,19 +62,62 @@ def get_context_data(self, **kwargs):
             data['plan']['amount'] = add_vat(data['plan']['amount'])
             context['actual_paid'] = data['plan']['amount']
 
+        if data.customer.invoice_settings.default_payment_method:
+            context['card_info'] = data.customer.invoice_settings.default_payment_method.card
+        else:
+            context['card_info'] = data.customer.default_source
+
         return context
 
 
 class SubscriptionView(StripeObjectMixin, NeverCacheMixin, DetailView):
     model = Subscription
     context_object_name = 'stripe'
+    needs_processing = False
```

Comment:
Nitpick, but this would be clearer as `needs_processing = None` and the below checks `if self.needs_processing is not None`. I was confused why a bool would be passed to `.update` on L115.